src/Sets/Zonotope.jl

```diff
@@ -329,30 +329,149 @@ function scale!(α::Real, Z::Zonotope)
     return Z
 end
 
-"""
-    reduce_order(Z::Zonotope, r::Union{Integer, Rational})
+# ==================================
+# Zonotope order reduction methods
+# ==================================
+
+# These methods split the given zonotope Z into two zonotopes, K and L, where
+# K contains the most "representative" generators and L contains the generators
+# that are reduced, Lred using a box overapproximation
+struct GIR05 <: AbstractReductionMethod end
+struct COMB03 <: AbstractReductionMethod end
+
+const _GIR05 = GIR05()
+const _COMB03 = COMB03()
```

Comment:
We do not need these. You can just pass the algorithm that you already have in the `_reduce_order` methods.